charts/victoria-logs-cluster/_index.md.gotmpl

```diff
@@ -20,37 +20,113 @@ tags:
 
 ## Prerequisites
 
-* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](https://docs.victoriametrics.com/helm/requirements/).
+Before installing this chart, ensure your environment meets the following requirements:
 
-* PV support on underlying infrastructure.
+* **Kubernetes cluster** - A running Kubernetes cluster with sufficient resources
+* **Helm** - Helm package manager installed and configured
+
+Additional requirements depend on your configuration:
+
+* **Persistent storage** - Required if you enable persistent volumes for data retention (enabled by default)
+* **kubectl** - Needed for cluster management and troubleshooting
+
+For installation instructions, refer to the official documentation:
+* [Installing Helm](https://helm.sh/docs/intro/install/)
+* [Installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ## Chart Details
 
-This chart will do the following:
+### VictoriaLogs Cluster
 
-* Rollout [VictoriaLogs cluster](https://docs.victoriametrics.com/victorialogs/cluster/).
-* (optional) Rollout [vector](https://vector.dev/) to collect logs from pods.
+When you install the chart, it creates three core services and an optional log collector:
```

Comment:
It is better to don't mention the number of components, since this number may change over time and this sentence may become inconsistent with the rest of the docs (this issue is repeated quite frequently in my practice). So it is better to replace the `three` with the `following`

```suggestion
When you install the chart, it creates the following core services and an optional log collector:
```